docs/run/nodes.md

```diff
@@ -1,48 +1,45 @@
+While npm v7 is not officially supported, you may still be able to get it to work. You will need to install the `node-pre-gyp` package globally. This is required until `node-webrtc` which [IPFS](../learn/glossary.md#ipfs) depends on [is upgraded](https://github.com/node-webrtc/node-webrtc/pull/694){:target="_blank"}.
 
-While npm v7 is not officially supported, you may be able to get it to work anyway, however you will need to have the `node-pre-gyp` package installed globally:
 ```bash
 npm install -g node-pre-gyp
 ```
-This is required until node-webrtc (which IPFS depends on) [is upgraded](https://github.com/node-webrtc/node-webrtc/pull/694).
-
-
-## **Installation**
 
+### 1. Install the CLI
 Open your console and install the CLI.
 
 ``` bash
 $ npm install -g @ceramicnetwork/cli
 ```
 
-
-## **Start the daemon**
-
-Next, start the Ceramic daemon.
+### 2. Start the Ceramic node
+This starts a Ceramic node with a standard configuration on the [Clay Testnet](../learn/networks.md#clay-testnet) at `https://localhost:7007`. Behind the scenes a properly configured [IPFS](../learn/glossary.md#ipfs) instance is started, then IPFS is used to create an instance of the Ceramic [JS Core Client](../clients/javascript) connected to the [Clay Testnet](../learn/networks.md#clay-testnet). Once Ceramic Core is running, an HTTP server is started that serves the Ceramic [HTTP API](../reference/http-api.md).
 
 ```bash
 $ ceramic daemon
 ```
 
-This command starts a Ceramic node with a standard cofiguration. By default the daemon runs a bundled IPFS instance. First, a properly configured IPFS instance is started. Then, IPFS is used to create an instance of Ceramic core. Once Ceramic core is running, an HTTP server is started that serves the Ceramic HTTP API. The node is now ready to accept requests from the local environment or remotely.
-
-### Using an external IPFS node
-
-It's also possible to run a separate service for IPFS using the `@ceramicnetwork/ipfs-daemon` npm package. 
+### 3. Use an external IPFS node
+By default the JS CLI starts a node with a bundled IPFS instance. If you would like to instead run a separate service for IPFS, use the `@ceramicnetwork/ipfs-daemon` npm package. 
 
+### 4. Configure a network
+By default the JS CLI starts a node on the Clay Testnet. If you would like to use a different network, you can specify this using the `--network` option. Currently, it is only possible to host nodes on the Clay Testnet. We will add the ability to run nodes on [Mainnet](../learn/networks.md#mainnet) in the near future.
 
-## **Network connectivity**
+### 5. Expose endpoints
+If you want your node and the streams created on it to be discoverable to the rest of the network, make sure the appropriate ports used by IPFS and the Ceramic HTTP API (by default this is port 7007) are publicly accessible. For example, if you are behind a router you may need to set up port forwarding. 
 
-### Expose ports
+### 6. Enable peer discovery

```

Comment:
Mention that this is a temporary measure.

Reply:
It's already mentioned in there.